redisinsight/api/src/modules/bulk-actions/models/bulk-action-summary.spec.ts

```diff
@@ -81,12 +81,30 @@ describe('BulkActionSummary', () => {
       expect(summary['errors']).toEqual(generateErrors(500));
     });
   });
+  describe('addKeys', () => {
+    it('should add keys', async () => {
+      expect(summary['keys']).toEqual([]);
+
+      summary.addKeys([Buffer.from('key1')]);
+
+      expect(summary['keys']).toEqual([Buffer.from('key1')]);
+
+      summary.addKeys([Buffer.from('key2'), Buffer.from('key3')]);
+
+      expect(summary['keys']).toEqual([
+        Buffer.from('key1'),
+        Buffer.from('key2'),
+        Buffer.from('key3'),
+      ]);
+    });
+  });
   describe('getOverview', () => {
     it('should get overview and clear errors', async () => {
       expect(summary['processed']).toEqual(0);
       expect(summary['succeed']).toEqual(0);
       expect(summary['failed']).toEqual(0);
       expect(summary['errors']).toEqual([]);
+      expect(summary['keys']).toEqual([]);
```

Comment:
do we still have keys property?

Reply:
Yeah, because of the other bulk action. I want to remove it from the API, but we need to re-implement the bulk import.

Reply:
It's not used tho, just not breaking the API.